src/collectors/mongodb/mongodb.py

```diff
@@ -147,16 +152,15 @@ def collect(self):
                     self.config['ssl'] = str_to_bool(self.config['ssl'])
 
                 if ReadPreference is None:
-                    conn = pymongo.Connection(
+                    conn = pymongo.MongoClient(
```

Comment:
What precipitated this change? Will users need to update pymongo to be compatible?